Tests/StreamChatTests/MockEnpointResponses/CurrentUser.json

```diff
@@ -12,6 +12,11 @@
   "online" : true,
   "name" : "Broken Waterfall",
   "secret_note" : "Anaking is Vader!",
+  "teams" : [
+      "RED",
+      "GREEN",
+      "BLUE"
+  ],
```

Comment:
According to the TeamDTO, it can also include users and channels? Or is that only in the DTO?

Reply:
Yes, however those properties are not fields of the payload gotten from response. We should handle the connections inside our codebase aka connect the team with the user (or channel) when assigning team/teams to those.